src/users/managers.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.base_user import BaseUserManager
+
+class CustomUserManager(BaseUserManager):
```

Comment:
what do you think about `Manager` responsibility?

Reply:
`A Manager is the interface through which database query operations are provided to Django models` 
Manager `database query` not a function to update object. Please research and use `Services` to do it

Reply:
oh sorry, i dont understand what u say before @@.I just create a manager.py to  shortcut  model file.and i override it in model file

Reply:
objects = CustomUserManager()

Reply:
![image](https://user-images.githubusercontent.com/38286740/88819922-dab3f280-d1ea-11ea-8402-8ec714303516.png)